docs/2.7.9/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst

```diff
@@ -20,7 +20,7 @@ Sequencer
 
 Sequencer HA depends on the chosen implementation. For example, when using a ledger such as `Hyperledger Fabric <../../../../canton/usermanual/domains/fabric.html>`_, HA is already set up. Multiple sequencer nodes must be deployed. 
 
-The domain returns multiple sequenced endpoints, any of which can be used to interact with the underlying ledger.
+The domain expose multiple sequencer endpoints, any of which can be used to submit sequencing requests or read events.
```

Comment:
Looks good, except here and in the other two changes below.

```suggestion
The domain exposes multiple sequencer endpoints, any of which can be used to submit sequencing requests or read events.
```

Reply:
:see_no_evil: